base-server/index.js

```diff
@@ -54,11 +55,18 @@ export class LoguxNotFoundError extends Error {
 
 function normalizeTypeCallbacks(name, callbacks) {
   if (callbacks && callbacks.accessAndProcess) {
-    callbacks.access = (...args) => {
+    callbacks.access = (ctx, action, meta) => {
       return wasNot403(async () => {
-        await callbacks.accessAndProcess(...args)
+        await callbacks.accessAndProcess(ctx, action, meta)
+        meta[SKIP_PROCESS] = true
       })
     }
+    callbacks.process = async (ctx, action, meta) => {
+      if (meta[SKIP_PROCESS]) {
```

Comment:
Nope. This will not work with different log storages (for instance, somebody may want to store log in database).

I recommend to use `ctx` since we re-use `ctx` between `access` and `process` calls.

Reply:
So we could just put the same flag in `ctx`?

Reply:
Yeap